frame/support/src/traits/messages.rs

```diff
@@ -220,3 +230,15 @@ where
 		E::footprint(O::get())
 	}
 }
+
+/// Provides information on paused queues.
+pub trait QueuePausedQuery<Origin> {
```

Comment:
As a design nit, I sometimes wonder if we should avoid creating a new trait like this when it can be easily expressed by `Convert<Origin, bool>` (cc @gpestana this is similar to my comment in https://github.com/paritytech/substrate/pull/13983#discussion_r1187519009 🙈).

No strong opinion myself, but I might personally lean into reusing `Convert` more and more.

Reply:
Interesting idea. Yes I also dont like having tons of trivial traits, just for naming.  

The only problem i see with `Convert` is on the implementor side. When seeing `impl<T: Config> Convert… for Pallet<T>`, it is not clear what it means. Maybe we can use type alias for that, or something similar.

Reply:
Maybe we could use the `frame_support::Get` trait instead? The type name is explicitly enough to use the getter trait and it things readable.

Reply:
Suppose we have `impl Get<bool> for Pallet<T>`.  
Now from the implementor side what does this mean? Anyone seeing this code will have no idea what it means.